heart/heartbeat.py

```diff
@@ -471,11 +490,11 @@ def on_msg(self, ct, msg):
 				d["address"] = str(self.addr)
 				d["port"], d["players_current"], d["players_max"] = struct.unpack("<HHH", msg[:6])
 				msg = msg[6:]
-				d["name"] = stripnul(msg[:30])
+				d["name"] = msg[:30].strip("\x00")
```

Comment:
I'd still prefer stripnul rather than what you've proposed - I'd like the NUL to actually terminate the string. Should save a bit of pain when serialising and deserialising the JSON.